### ISSUE TITLE
Intro to Voluspa: Suggest making your reasoning for contractions explicit

### ISSUE THREAD

**Issue by jgnewman** [martensas/edda#34]
> At many places superfluous pronouns have been removed, and almost everywhere ek ‘I’, es (particle) ‘which, that, where, when’, and es (3rd sg. pres. ind. of vesa ‘to be’) have been contracted to ’k and ’s, respectively.

When we chatted previously, you mentioned that this was done for metrical reasons. I suggest calling that out here as well.

**Comment by martensas** [comment]
Yes, I am planning on moving all of this to the general introduction since it applies to all poetry edited in this book. I'll close this issue when I do that (probably later today).
